main.yml

```diff
@@ -0,0 +1,42 @@
+name: build
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: [3.2, 3.3, 3.4, 3.5]
```

Comment:
these'll need to be the golang versions: `[1.10, 1.11, 1.12, 1.13]` if you want to use them with `actions/setup-go@v1` as `.travis.yml` is.  I can add more recent golang versions in later PR.